packages/tenant-readmodel-writer/test/converterV1.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  TenantMail,
+  TenantMailKind,
+  Tenant,
+  TenantAttribute,
+  TenantAttributeV1,
+  TenantFeature,
+  TenantFeatureV1,
+  TenantKind,
+  TenantKindV1,
+  TenantMailKindV1,
+  TenantMailV1,
+  TenantRevoker,
+  TenantRevokerV1,
+  TenantV1,
+  TenantVerifier,
+  TenantVerifierV1,
+  tenantMailKind,
+  tenantKind,
+  tenantAttributeType,
+  TenantUnitTypeV1,
+  tenantUnitType,
+  TenantUnitType,
+  dateToBigInt,
+} from "pagopa-interop-models";
+import { match } from "ts-pattern";
+
+export function toFeatureV1(feature: TenantFeature): TenantFeatureV1 {
+  return match<TenantFeature, TenantFeatureV1>(feature)
+    .with({ type: "PersistentCertifier" }, (feature) => ({
+      sealedValue: {
+        oneofKind: "certifier",
+        certifier: {
+          certifierId: feature.certifierId,
+        },
+      },
+    }))
+    .exhaustive();
+}
+
+export function toTenantVerifierV1(verifier: TenantVerifier): TenantVerifierV1 {
+  return {
+    id: verifier.id,
+    verificationDate: dateToBigInt(verifier.verificationDate),
+    expirationDate: verifier.expirationDate
+      ? dateToBigInt(verifier.expirationDate)
+      : undefined,
+    extensionDate: verifier.extensionDate
+      ? dateToBigInt(verifier.extensionDate)
+      : undefined,
```

Comment:
The `undefined` case is handled inside `dataToBigInt`. There is no need to add the check.